cmd/tailscale/cli/configure-synology.go

```diff
@@ -18,9 +18,23 @@ import (
 	"tailscale.com/version/distro"
 )
 
+// configureHostCmd is the "tailscale configure-host" command which
+// was once used to configure Synology devices, but is now a redirect
```

Comment:
now a compatibility alias for "tailscale configure synology"

Reply:
Done